FACBackEnd/jest-dynamodb-config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
You should add a comment that describes the function of this file, and maybe a link to the website that provided you with this code